src/axolotl/core/trainer_builder.py

```diff
@@ -898,10 +898,6 @@ def get_callbacks(self):
         ):
             callbacks.append(SaveBetterTransformerModelCallback())
 
-        if self.cfg.use_wandb:
-            callbacks.append(
-                SaveAxolotlConfigtoWandBCallback(self.cfg.axolotl_config_path)
```

Comment:
is there a reason this is removed

Reply:
It's duplicated from elsewhere in the code base that already adds this callback.

Reply:
Hm, I checked and found the offending section. https://github.com/OpenAccess-AI-Collective/axolotl/blob/03b0066d478dee8351d446e160598b5196c0dede/src/axolotl/core/trainer_builder.py#L850-L857

Would it be better to delete this one instead to consolidate all the callbacks?

Reply:
not really, b/c you would need to duplicate this again for DPO if you removed it from the base class.

Reply:
makes sense!